crates/sui-protocol-config/src/lib.rs

```diff
@@ -1097,6 +1098,12 @@ pub struct ProtocolConfig {
     /// The max number of consensus rounds a transaction can be deferred due to shared object congestion.
     /// Transactions will be cancelled after this many rounds.
     max_deferral_rounds_for_congestion_control: Option<u64>,
+
+    /// Minimum interval of commit timestamps between consecutive checkpoints.
+    min_checkpoint_interval_ms: Option<u64>,
+
+    /// Version number to use for version_specific_data in `CheckpointSummary`.
+    checkpoint_summary_version_specific_data: Option<u64>,
```

Comment:
Nit: would this be an enum in feature flag? (to get rid of all the unimplemented!()).

Reply:
I originally had it as a feature flag and @mystenmark asked me to use this style instead, other "version" settings in the protocol config also just use ints e.g. `move_binary_format_version`, `gas_model_version`, `execution_version`